examples/language-modeling/run_mlm_flax.py

```diff
@@ -134,6 +134,15 @@ class DataTrainingArguments:
     overwrite_cache: bool = field(
         default=False, metadata={"help": "Overwrite the cached training and evaluation sets"}
     )
+    validation_split_percentage: Optional[int] = field(
+        default=5,
+        metadata={
+            "help": "The percentage of the train set used as validation set in case there's no validation split"
+        },
+    )
+    test_split_percentage: Optional[int] = field(
+        default=5, metadata={"help": "The percentage of the train set used as test set in case there's no test split"}
+    )
```

Comment:
This has no link to the PR so I would like those to be discussed in another PR. This is a bit half-baked in the sense that:
- If it's added here, it should also be added in the PyTorch example.
- Why introduce an API for a test set that is not used? 

So I'd vote for removing that and focus on the `from_pretrained`/initialization here, we can add those argument for all new scripts and discuss the API in a new PR